README.md

```diff
@@ -122,13 +122,79 @@ curl -H "authorization: Bearer executor-secret-key" -H "Content-Type: applicatio
 docker compose down
 ```
 
-## Endpoints
-
-TODO: Document each endpoint + CURL examples
+## API Endpoints
+
+| Method | Endpoint | Description | Params | 
+|--------|----------|-------------| ------ |
+| GET |`/v1/runtimes/{runtimeId}/logs`| Get live stream of logs of a runtime | [JSON](#v1runtimesruntimeidlogs) |
+| POST |`/v1/runtimes`| Create a new runtime server | [JSON](#v1runtimes) |
+| GET |`/v1/runtimes`| List currently active runtimes | X |
+| GET |`/v1/runtimes/{runtimeId}`| Get a runtime by its ID | [JSON](#v1runtimesruntimeid) |
+| DELETE |`/v1/runtimes/{runtimeId}`| Delete a runtime | [JSON](#v1runtimesruntimeid) |
+| POST |`/v1/runtimes/{runtimeId}/executions`| Create an execution | [JSON](#v1runtimesruntimeidexecutions) |
+| GET |`/v1/health`| Get health status of host machine and runtimes | X |
+
+### /v1/runtimes/{runtimeId}/logs
+| Param | Type | Description | Required |
```

Comment:
We could extend table by 1 more column, `default`. If required=No, then showing default is valuable